bridge/docs/bridging.md

```diff
@@ -34,3 +34,94 @@ Again, a withdrawfee of 1 TFT will be taken, so make sure you send a larger amou
 The amount withdrawn from TF Chain will be sent to your Stellar wallet.
```

Comment:
Om line 31, just above
Browse to https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftfchain.grid.tf#/extrinsics (for mainnet)